model/index.js

```diff
@@ -45,6 +45,15 @@ module.exports = yeoman.generators.Base.extend({
   loadModels: actions.loadModels,
 
   addNullDataSourceItem: actions.addNullDataSourceItem,
+  
+  checkForDatasource: function() {
+    if (!this.hasDatasources) {
+      var warning = chalk.red('Warning: No data sources available ' +
+        'to attach model.');
```

Comment:
Can we please extend the message and tell the user what are the implications? What should they expect to happen? This way it's hard to tell whether how serious it is that there are not data sources available.